lib/util.js

```diff
@@ -38,35 +38,23 @@ if (typeof global.SIMD === 'object' && global.SIMD !== null) {
 
   const SIMD = global.SIMD;  // Pacify eslint.
 
-  if (typeof SIMD.Bool16x8 === 'function')
-    simdFormatters.set(SIMD.Bool16x8, make(SIMD.Bool16x8.extractLane, 8));
-
-  if (typeof SIMD.Bool32x4 === 'function')
-    simdFormatters.set(SIMD.Bool32x4, make(SIMD.Bool32x4.extractLane, 4));
-
-  if (typeof SIMD.Bool8x16 === 'function')
-    simdFormatters.set(SIMD.Bool8x16, make(SIMD.Bool8x16.extractLane, 16));
-
-  if (typeof SIMD.Float32x4 === 'function')
-    simdFormatters.set(SIMD.Float32x4, make(SIMD.Float32x4.extractLane, 4));
-
-  if (typeof SIMD.Int16x8 === 'function')
-    simdFormatters.set(SIMD.Int16x8, make(SIMD.Int16x8.extractLane, 8));
-
-  if (typeof SIMD.Int32x4 === 'function')
-    simdFormatters.set(SIMD.Int32x4, make(SIMD.Int32x4.extractLane, 4));
-
-  if (typeof SIMD.Int8x16 === 'function')
-    simdFormatters.set(SIMD.Int8x16, make(SIMD.Int8x16.extractLane, 16));
-
-  if (typeof SIMD.Uint16x8 === 'function')
-    simdFormatters.set(SIMD.Uint16x8, make(SIMD.Uint16x8.extractLane, 8));
-
-  if (typeof SIMD.Uint32x4 === 'function')
-    simdFormatters.set(SIMD.Uint32x4, make(SIMD.Uint32x4.extractLane, 4));
+  const countPerType = {
```

Comment:
yeah as in `new Map()`
@targos `simdFormatters` is kind of the same case so consistency(?) :P